jobma/models.py

```diff
@@ -15,7 +15,7 @@ class Job(AuditableModel):
     job_code = models.TextField()
     job_title = models.TextField()
     interview_template_id = models.IntegerField()
-    run = models.ForeignKey(BootcampRun, on_delete=models.CASCADE, unique=True)
+    run = models.ForeignKey(BootcampRun, on_delete=models.CASCADE)
```

Comment:
It conflicts with https://github.com/mitodl/bootcamp-ecommerce/pull/738. Secondly, when the error says:
```
jobma.Job.run: (fields.W342) Setting unique=True on a ForeignKey has the same effect as using a OneToOneField.
	HINT: ForeignKey(unique=True) is usually better served by a OneToOneField.
```
It asks to make it an OneToOneField rather than removing the unique constraint, corrupting the whole logic.

Reply:
Actually, putting a unique constraint is not recommended on a ForeignKey so either we need to convert field to 1-1 or remove the unique constraint. 
I am not sure, what are the actual requirements for making a unique foreign key earlier. @noisecapella can you please suggest about the reasons in the [PR](https://github.com/mitodl/bootcamp-ecommerce/pull/738)?

Reply:
I would keep the unique constraint and use a `OneToOneField` as @HamzaIbnFarooq suggested. I think a `OneToOneField` would have an identical database schema as a `ForeignKey` with a unique constraint so it should only require code changes, hopefully minor ones

Reply:
OneToOneField always carries a "unique" constraint with it so no need to specify it